esx_service/cli/vmdkops_admin.py

```diff
@@ -591,7 +591,8 @@ def get_service_status():
         if output[2] == "not":
             return NOT_RUNNING_STATUS
 
-        pid = output[3].split("=")[1]
```

Comment:
This was the error:

File "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py", line 594, in get_service_status
pid = output[3].split("=")[1]
TypeError: a bytes-like object is required, not 'str'